src/storage/index/LookupBaseProcessor-inl.h

```diff
@@ -319,8 +321,8 @@ std::unique_ptr<IndexOutputNode<IndexID>> LookupBaseProcessor<REQ, RESP>::buildP
   auto indexId = ctx.get_index_id();
   auto colHints = ctx.get_column_hints();
 
-  auto indexScan = std::make_unique<IndexScanNode<IndexID>>(
-      context_.get(), indexId, std::move(colHints), limit_);
+  auto indexScan =
+      std::make_unique<IndexScanNode<IndexID>>(context_.get(), indexId, std::move(colHints));
```

Comment:
Why doesn't this plan need `limit`?

Reply:
> Why doesn't this plan need `limit`?

if FilterNode exists. will check the limit in FilterNode. skip the IndexScanNode.